packages/p2p-v2/src/components/BuySellForm/BuySellForm.tsx

```diff
@@ -79,50 +104,66 @@ const BuySellForm = ({
-        const payload = {
+        const payload: TPayload = {
             advert_id: id,
             amount: Number(getValues('amount')),
         };
         if (rateValue) {
             payload.rate = rateValue;
         }
+
+        if (isBuy && selectedPaymentMethods.length) {
+            payload.payment_method_ids = selectedPaymentMethods;
+        }
+
+        if (isBuy && !selectedPaymentMethods.length) {
+            payload.payment_info = getValues('bank_details');
+        }
+
         mutate(payload);
     };
 
     const calculatedRate = removeTrailingZeros(roundOffDecimal(effectiveRate, setDecimalPlaces(effectiveRate, 6)));
     const initialAmount = removeTrailingZeros((min_order_amount_limit * Number(calculatedRate)).toString());
 
+    const onSelectPaymentMethodCard = (paymentMethodId: number) => {
+        if (selectedPaymentMethods.includes(paymentMethodId)) {
+            setSelectedPaymentMethods(selectedPaymentMethods.filter(method => method !== paymentMethodId));
+        } else {
+            setSelectedPaymentMethods([...selectedPaymentMethods, paymentMethodId]);
+        }
+    };
+
     return (
-        <Modal
-            className='p2p-v2-buy-sell-form'
-            isOpen={isModalOpen}
-            onRequestClose={onRequestClose}
-            style={{ ...customStyles, overlay: { ...customStyles.overlay, top: `${isMobile ? '16rem' : 0}` } }}
-        >
-            <BuySellFormHeader
-                currency={account_currency}
-                onClickBack={isMobile ? onRequestClose : undefined}
-                type={type}
-            />
-            <Divider />
-            {rate_type === RATE_TYPE.FLOAT && !shouldDisableField && (
-                <div className='px-[2.4rem] mt-[2.4rem]'>
-                    <InlineMessage variant='info'>
-                        <Text size={isMobile ? 'xs' : '2xs'}>
-                            {`If the market rate changes from the rate shown here, we won't be able to process your order.`}
-                        </Text>
-                    </InlineMessage>
-                </div>
-            )}
-            <form onSubmit={handleSubmit(onSubmit)}>
+        <form onSubmit={handleSubmit(onSubmit)}>
+            <BuySellFormDisplayWrapper
+                accountCurrency={account_currency}
+                isBuy={isBuy}
+                isModalOpen={isModalOpen}
+                isValid={isValid}
+                onRequestClose={onRequestClose}
+                onSubmit={onSubmit}
+            >
+                {rate_type === RATE_TYPE.FLOAT && !shouldDisableField && (
+                    <div className='px-[2.4rem] mt-[2.4rem]'>
+                        <InlineMessage variant='info'>
+                            <Text size={isMobile ? 'xs' : '2xs'}>
+                                {
+                                    'If the market rate changes from the rate shown here, we won’t be able to process your order.'
+                                }
```

Comment:
we dont need the brackets right
```suggestion
                                
                                    'If the market rate changes from the rate shown here, we won’t be able to process your order.'
                                
```

Reply:
doesn't work without the brackets.

Reply:
you need to remove the quotation marks as well and have the `»`’` not `'` as apostrophe